RedHatInsights/insights-inventory-frontend
# fix(ESSNTL-4937): Display group name even when can't see hosts

Fixes https://issues.redhat.com/browse/ESSNTL-4937?focusedId=22605892&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-22605892.

## How to test

Have an account without `inventory:hosts:read`, but having `inventory:groups:read` - enough to open a group details page. Check that the page title has the group name and not id.

## Screenshots

![image](https://github.com/RedHatInsights/insights-inventory-frontend/assets/31385370/fdebc232-9941-410f-9d99-eae52a17115f)
